File: java/src/televentas/NotificadorEmailConsola.java
package televentas;

public class NotificadorEmailConsola implements NotificadorEmail {

    @Override
    public void enviar(String destinatario, String asunto, String contenido) {
        System.out.println();
        System.out.println("--- CORREO ENVIADO ---");
        System.out.println("Para: " + destinatario);
        System.out.println("Asunto: " + asunto);
        System.out.println("Contenido: " + contenido);
        System.out.println("----------------------");
        System.out.println();
    }
}

File: javascript/src/televentas/notificadorEmailConsola.js
import { NotificadorEmail } from "./notificadorEmail.js";

export class NotificadorEmailConsola extends NotificadorEmail {
  constructor() {
    super();
  }

  enviar(destinatario, asunto, contenido) {
    console.log();
    console.log("--- CORREO ENVIADO ---");
    console.log(`Para: ${destinatario}`);
    console.log(`Asunto: ${asunto}`);
    console.log(`Contenido: ${contenido}`);
    console.log("----------------------");
    console.log();
  }
}
